### PR TITLE
Fetch a single document.

### PR DESCRIPTION
**Pull request by mattgrill**
## Issue

Fetching all the schema that OpenAPI provides is a bit much. This reduces that.

## Testing instructions

- Make sure `/node/add/recipe` still works.

### CODE REVIEW

File: src/reducers/content.js [hunk 1]
```diff
@@ -47,7 +47,10 @@ export default (state = initialState, action) => {
     case SCHEMA_LOADED: {
       return {
         ...state,
-        schema: action.payload.schema,
+        schema: {
```
Comment (dawehner) [inline src/reducers/content.js]: What should we do with schema loaded beforehand?

Comment (dawehner) [reply src/reducers/content.js]: Is it okay to throw that information away?

File: src/actions/content.js [hunk 2]
```diff
@@ -10,22 +10,26 @@ import { MESSAGE_ERROR } from '../constants/messages';
...
+export const requestSchema = ({ entityTypeId, bundle }) => ({
   type: SCHEMA_REQUESTED,
-  payload: {},
+  payload: { entityTypeId, bundle },
 });
 
 export const SCHEMA_LOADED = 'SCHEMA_LOADED';
-function* loadSchema() {
+function* loadSchema(action) {
+  const { entityTypeId, bundle } = action.payload;
```
Comment (dawehner) [inline src/actions/content.js]: I thought about whether we want to cache schema for now, but I think till we are aware of all the potential complexities around that, not caching is the most sane approach.

Comment (mattgrill) [reply src/actions/content.js]: Yes. Ok. So I think this is good?

Comment (dawehner) [reply src/actions/content.js]: Yes!

**Comment by mattgrill** [comment]
🙏

**Pull request merged**